.ci/containers/terraform-vcr-tester/run_vcr_tests.sh

```diff
@@ -111,44 +111,56 @@ NEWLINE=$'\n'
 set +e
 TEST_RESULTS_URL="http://ci-oss.hashicorp.engineering/app/rest/testOccurrences?locator=build:${ID}"
 while [ -n "${TEST_RESULTS_URL}" ]; do
-	echo $TEST_RESULTS_URL
-	curl \
-		--header "Accept: application/json" \
-		--header "Authorization: Bearer $TEAMCITY_TOKEN" \
-		--output tests.json \
-		-L \
-		"${TEST_RESULTS_URL}"
-
-	# Alert on tests that failed without running anything
-	if [[ $(cat tests.json | jq -r '.count') == "0" ]]; then
-		echo "Job failed without failing tests"
-		update_status "${build_url}" "failure"
-		# exit 0 because this script didn't have an error; the failure
-		# is reported via the Github Status API
-		exit 0
-	fi
-
-	RUN_TESTS+=$(cat tests.json | jq -r '.testOccurrence | map(select(.status != "UNKNOWN"))| map(.name) | .[]')
-	FAILED_TESTS+=$(cat tests.json | jq -r '.testOccurrence | map(select(.status == "FAILURE")) | map(.name) | join("|")')
-	next_href=$(cat tests.json | jq -r '.nextHref')
-	if [[ $next_href == "null" ]]; then
-		break
-	else
-		TEST_RESULTS_URL="http://ci-oss.hashicorp.engineering$next_href"
-	fi
+  echo $TEST_RESULTS_URL
+  curl \
+    --header "Accept: application/json" \
+    --header "Authorization: Bearer $TEAMCITY_TOKEN" \
+    --output tests.json \
+    -L \
+    "${TEST_RESULTS_URL}"
+
+  # Alert on tests that failed without running anything
+  if [[ $(cat tests.json | jq -r '.count') == "0" ]]; then
+    echo "Job failed without failing tests"
+    update_status "${build_url}" "failure"
+    # exit 0 because this script didn't have an error; the failure
+    # is reported via the Github Status API
+    exit 0
+  fi
+
+  NEW_RUN_TESTS=$(cat tests.json | jq -r '.testOccurrence | map(select(.status != "UNKNOWN"))| map(.name) | .[]')
+  NEW_FAILED_TESTS=$(cat tests.json | jq -r '.testOccurrence | map(select(.status == "FAILURE")) | map(.name) | join("|")')
+  if [ -n "$NEW_RUN_TESTS" ] && [ -n "$RUN_TESTS" ]
+  then
+    RUN_TESTS+="|"
```

Comment:
this should use `${NEWLINE}` instead; `RUN_TESTS` is a list of one test name per line.

Reply:
I think I got it $'\n'